README.md

```diff
@@ -1,8 +1,11 @@
-[![Build Status](https://app.bitrise.io/app/d80a106bb67fa6e8/status.svg?token=xnyYwDyHqw773UCliHgd4g&branch=develop)](https://app.bitrise.io/app/d80a106bb67fa6e8)
-[![Maintainability](https://api.codeclimate.com/v1/badges/0265f119ef5164d3ee70/maintainability)](https://codeclimate.com/github/bykovme/nswallet/maintainability)
+[![NS Wallet](https://github.com/bykovme/nswallet/blob/update/badges/banner.png?raw=true)](https://nswallet.com/)
```

Comment:
Change description on the image to "Free Offline Multi-platform Password Manager"

Reply:
Changed. Look at [commit 70287d7](https://github.com/bykovme/nswallet/commit/70287d7ba61d32103054e0e084d16e4ba4bb8a45).